src/useAnimate.ts

```diff
@@ -32,22 +35,26 @@ export default function useAnimate(
     [],
   );
 
-  const play = (isPlay: boolean) => {
-    setStyle({
-      ...(isPlay ? end : start),
-      transition,
-    });
+  const play = React.useCallback(
+    (isPlay: boolean) => {
+      setStyle({
+        ...(isPlay ? end : start),
+        transition,
+      });
 
-    setIsPlaying(isPlay);
+      setIsPlaying(true);
```

Comment:
Thanks for the PR :) why it’s always true here?

Reply:
Cause even if we do `play(false)` animation starts and ends after Timeout

Reply:
And I've added `setIsPlaying(false);` in timeout callback

Reply:
But yeah.... something is still wrong here (some conditions need to be changed) :)

Reply:
hmmm this may break the usage, i will investigate it bit more. Thanks tho :)